packages/@glimmer/integration-tests/test/compiler/compile-options-test.ts

```diff
@@ -26,6 +26,40 @@ module('[glimmer-compiler] precompile', ({ test }) => {
     assert.strictEqual(wire.moduleName, 'my/module-name', 'Template has correct meta');
   });
 
+  test('lexicalScope is used if present', (assert) => {
```

Comment:
can we add a test (or does one already exist somewhere else?) for `<x.someProperty />`?